BranchSDK/src/BranchIO/RequestManager.cpp

```diff
@@ -132,7 +132,7 @@ void RequestManager::run() {
 
 RequestManager::RequestTask::RequestTask(
     RequestManager& manager,
-    const Event& event,
+    const Event::Event& event,
```

Comment:
We don't generally use `using` statements, but this looks weird.   I know it is necessary, just makes me re-think the name of the class and folder structure.